From 38898c58b9cc93c3bc3196a23d2cad2914913fba Mon Sep 17 00:00:00 2001
From: Varun <vyaparla@habana.ai>
Date: Thu, 19 May 2022 16:46:07 +0300
Subject: [PATCH] [SW-87241] sec: preboot image verification failure

This patch add changes to fix the preboot image verification
failure by ppboot due to stack overflow. Stack size in ppboot
is kept as 512B and it is not sufficient. This patch increases
stack size to 2048B.

Change-Id: Ibdc95ab5717b28545c8aef846cd7bde90d485517
Signed-off-by: Varun <vyaparla@habana.ai>
---

diff --git a/arch/gaudi2/arc/ppboot.lds.S b/arch/gaudi2/arc/ppboot.lds.S
index f70569c..951fe4d 100644
--- a/arch/gaudi2/arc/ppboot.lds.S
+++ b/arch/gaudi2/arc/ppboot.lds.S
@@ -63,7 +63,7 @@
 	.stack : {
 		. = ALIGN(8);
 		__stack = . ;
-		. = . + 512 ;
+		. = . + 2048 ;
 		__stack_top = . ;
 	} > REGION_STACK
 
